### ISSUE TITLE
cannot load such file -- rszr/image_processing (LoadError)

### ISSUE THREAD

**Issue by KevinColemanInc** [mtgrosser/rszr#6]
ruby 2.6

```
5: from /Users/kevin/projects/rails/config/initializers/rszr.rb:1:in `<top (required)>'
4: from /Users/kevin/.rvm/gems/ruby-2.6.6/gems/activesupport-5.2.4.4/lib/active_support/dependencies.rb:291:in `require'
3: from /Users/kevin/.rvm/gems/ruby-2.6.6/gems/activesupport-5.2.4.4/lib/active_support/dependencies.rb:257:in `load_dependency'
2: from /Users/kevin/.rvm/gems/ruby-2.6.6/gems/activesupport-5.2.4.4/lib/active_support/dependencies.rb:291:in `block in require'
1: from /Users/kevin/.rvm/gems/ruby-2.6.6/gems/skylight-core-4.3.1/lib/skylight/core/probes.rb:118:in `require'
/Users/kevin/.rvm/gems/ruby-2.6.6/gems/skylight-core-4.3.1/lib/skylight/core/probes.rb:118:in `require': cannot load such file -- rszr/image_processing (LoadError)
```

**Comment by mtgrosser** [comment]
Support for `image_processing` is currently only in master, I'll release a new version shortly.

**Comment by mtgrosser** [comment]
Resolved since v0.7.1
